packages/eas-cli/src/commands/metadata/configure.ts

```diff
@@ -0,0 +1,85 @@
+import { getConfig } from '@expo/config';
```

Comment:
metadata name is not descriptive, without looking at docs users will have no idea what it means, also metadata world on its own is not clear what it will do( build,submit,update are both verbs and nouns metadata on it's own is not clear)

few suggestions
- `submit:metadata:update` and `submit:metadata:configure`
- `metadata:appstore:update`/`metadata:playstore:update`/`metadata:appstore:configure`/`metadata:playstore:configure` (this might be better alternative to platform flag, because we might want to support multiple stores per platform

Reply:
1. That seems ok to me, would you also be ok to add short aliases with the current ones? e.g. `submit:metadata:update`, which aliases to `submit:metadata` or even `metadata`?
2. Right now, iOS doesn't allow any other app store and Android doesn't have a good official 2nd store without going into shady websites, so I don't think we want to support multiple stores per platform (just the official ones). But let me check with Evan first!

Reply:
Ad:1 I'm not suggesting a change because I think my option is better, but because the shorter version is actively confusing, so adding aliases would defeat that purpose.
Ad: 2 I was thinking about android stores e.g. amazon one

Reply:
As discussed, keeping this open for the next store to implement.